src/makeCalculator.js

```diff
@@ -37,7 +37,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    operate(callback, value) {
+      this.result = callback(this.result, value);
+
+      return this;
+    },
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+    add(a, b) {
+      return a + b;
+    },
+    subtract(a, b) {
+      return a - b;
+    },
+    multiply(a, b) {
+      return a * b;
+    },
+    divide(a, b) {
+      return a / b;
+    },
+  };
```

Comment:
Maybe it'll be better to write "return calculator", but I'm not sure
![image](https://user-images.githubusercontent.com/110456763/191038287-e212d359-2061-47ad-8a2a-62cbb2c43e4d.png)